firmware/controllers/engine_cycle/rpm_calculator.cpp

```diff
@@ -133,7 +133,7 @@ bool RpmCalculator::checkIfSpinning(efitick_t nowNt) const {
 void RpmCalculator::assignRpmValue(float floatRpmValue) {
 	previousRpmValue = rpmValue;
 	// we still persist integer RPM! todo: figure out the next steps
-	rpmValue = floatRpmValue;
+	rpmValue = floatRpmValue + 0.5f;
```

Comment:
please add a comment "We believe that rounding is better than truncating" or something along these lines?

Reply:
I mean, rounding is always better than truncating (when operating with real world physical quantities, at least), but comment added/modified.